x/incentives/keeper/evm_hooks.go

```diff
@@ -15,6 +16,15 @@ var _ evmtypes.EvmHooks = Hooks{}
 // interaction with an incentivized contract, the participants's GasUsed is
 // added to its gasMeter.
 func (h Hooks) PostTxProcessing(ctx sdk.Context, participant common.Address, contract *common.Address, receipt *ethtypes.Receipt) error {
+	// check if the Incentives are globally enabled
+	params := h.k.GetParams(ctx)
+	if !params.EnableIncentives {
+		return sdkerrors.Wrap(
+			types.ErrInternalIncentive,
+			"incentives are currently disabled by governance",
+		)
```

Comment:
we don't want the transaction to be reverted, so we need to return `nil`
```suggestion
		return ni
```

Reply:
ah nice thank you!